pom.xml

```diff
@@ -273,8 +274,7 @@
     <maven.resources.plugin.version>3.3.1</maven.resources.plugin.version>
     <maven.shade.plugin.version>3.6.0</maven.shade.plugin.version>
     <maven.source.plugin.version>3.3.1</maven.source.plugin.version>
-    <!-- do not upgrade as jpms issue with ee10 annotations -->
-    <maven.surefire.plugin.version>3.3.0</maven.surefire.plugin.version>
+    <maven.surefire.plugin.version>3.5.3</maven.surefire.plugin.version>
```

Comment:
I'm guessing this JPMS issue was resolved then?

Reply:
sure :)